blenderproc/python/utility/InstallUtility.py

```diff
@@ -204,7 +204,7 @@ def __call__(self, block_num, block_size, total_size):
                 # rename the blender folder to better fit our existing scheme
                 for folder in os.listdir(blender_install_path):
                     if os.path.isdir(os.path.join(blender_install_path, folder)) and \
-                            folder.startswith("blender-" + major_version):
+                            folder.startswith(f"blender-{major_version}.{minor_version}"):
                         os.rename(os.path.join(blender_install_path, folder),
                                   os.path.join(blender_install_path, blender_version))
```

Comment:
Does this work on all platforms?

Reply:
Good question, it should based on the code. Can you maybe test it on mac, then I am gonna check windows?